templates/views/newshub.pug

```diff
@@ -3,75 +3,76 @@ extends ../layouts/default
 block content
   .newshub_container
     .newshub
-      
-      // sidebar left
-      if content.sidebarLeft
-        #sidebar_left(style="background:#" + content.sidebarLeft.backgroundcolor)
-          a(href=content.sidebarLeft.linkUrl)
-            img(src=content.sidebarLeft.imageUrl alt="")
-
-      // sidebar right
-      if content.sidebarRight
-        #sidebar_right(style="background:#" + content.sidebarRight.backgroundcolor)
-          a(href=content.sidebarRight.linkUrl)
-            img(src=content.sidebarRight.imageUrl alt="")
-
       // top bar
       if content.topbar
         .announcement
           a(href=content.topbar.linkUrl)
             div(style="background:#" + content.topbar.backgroundcolor)
               | !{ content.topbar.paragraph }
+      .newshub_flex
+        // sidebar left
+        if content.sidebarLeft
+          #sidebar_left(style="background:#" + content.sidebarLeft.backgroundcolor)
+            a(href=content.sidebarLeft.linkUrl)
+              img(src=content.sidebarLeft.imageUrl alt="")
 
-      // normalArticles 
-      .column1
-      
-        for a in content.normalArticles
+        // sidebar right
+        if content.sidebarRight
+          #sidebar_right(style="background:#" + content.sidebarRight.backgroundcolor)
+            a(href=content.sidebarRight.linkUrl)
+              img(src=content.sidebarRight.imageUrl alt="")
```

Comment:
Do you need that alt? If yes, fill it, if not, don't use it

Reply:
will remove in next release, lets just push this out